Modules/Core/Common/include/itkBoolean.h

```diff
@@ -0,0 +1,54 @@
+/*=========================================================================
+ *
+ *  Copyright NumFOCUS
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0.txt
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *=========================================================================*/
+#ifndef itkBoolean_h
+#define itkBoolean_h
+
+namespace itk
+{
+
+/**
+ * \class Boolean
+ *
+ * \brief An alternatitve to bool for addtional thread safety
```

Comment:
Two more typo's here: "alternatitve" and "addtional". But no big deal, as you removed so many typo's already, recently. 😄

Reply:
Fixed with the most recent push.